### DISCUSSION TITLE
Filtering dropdown navbar

### DISCUSSION THREAD

**Discussion by AutumnFoto** [nss-day-cohort-47/C47-Discussions#56] [Help Me] [answered]
Your Name

Autumn Fotopoulos

Have you searched the other help tickets? If not, do that first (Duplicates asks for help will be shamed ceaselessly)
yes 

If there are similar issues, make sure to give us their number/link to them and tell how yours is different
Describe your issue

I can create a dropdown menu in the navbar, I can create switch or else if statements for the filtering, but confused on how to actually link it all together where you click the drop down menu buttons and they filter the data on the page. 

List any error messages that appear
n/a 

What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
https://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_filters_dropdown&stacked=h
https://www.w3schools.com/howto/howto_js_dropdown.asp
https://www.youtube.com/watch?v=kzFJ7St_ma8

What have you tried? Tell us everything you have tried.

tried making dropdown in html, tried dropdown in js and then doing if esle statements, but I honestly cant figure it out to make the clicks work on the dropdown and cant find anything on the web to help.

**Comment by Chadwell-Clark** [comment]
You probably want to use the "change" event rather than "click"
Then you can get the select option in the dropdown with event.target.value

**Reply by AutumnFoto** [reply]
okay thank you!!

**Comment by AutumnFoto** [comment, marked as answer]
this is what I ended up doing 
![dropdown](https://user-images.githubusercontent.com/67810121/110997725-6e35d000-8343-11eb-82af-e645c8ca0b56.png)
![2021-03-12 14_58_19-main js - everything-is-awesome-AutumnFoto - Visual Studio Code](https://user-images.githubusercontent.com/67810121/110997842-96253380-8343-11eb-8b05-bb281b09b0e5.png)

**Reply by Chadwell-Clark** [reply]
I wrote a switch statement that was about 35 lines of code for  the material dropdown event handler...ugh
It could have been three...

**Reply by AutumnFoto** [reply]
but hey! you learned it a different way, Red was telling some of us to try switch statements and I know they are like if else statements but I couldnt wrap my head around it lol. Im going to checkout your code with the switch statement!

**Reply by Chadwell-Clark** [reply]
I cloned the repository and because I had accidentally messed up the song workshop  by not making a branch and then pushing main, I am leery of doing that again so I worked on my own fork and removed the origin. I need to figure out the right way to do this. I will push mine up to the Cohort 47 Github now.
